crates/telio-sockets/src/protector/apple.rs

```diff
@@ -55,35 +57,33 @@ pub(crate) fn bind_to_tun(sock: NativeSocket, tunnel_interface: u64) -> io::Resu
 }
 
 pub(crate) fn bind(interface_index: u32, socket: i32) -> io::Result<()> {
+    let nz_index = std::num::NonZeroU32::new(interface_index);
```

Comment:
Just for my understanding. Why is a zero index problematic ? Is it because the system would choose interface automatically ?

Reply:
I do not know the actual reason, but it could be that 0 means _no interface_ or it is a `lo`.

Reply:
But I just used it as in an example ...